app/components/side-bar.hbs

```diff
@@ -3,7 +3,7 @@
   <FaIcon @icon="caret-down" @fixedWidth={{true}} />  </button>
   <div class="sidebar__dropdown-container {{if @showDropDown "show" "close"}}">
     {{#each @taskStatusList as |status|}}
-      <a href="#" {{on "click" (fn @toggleTasks status)}}>{{status}}</a>
+      <a href="#" {{on "click" (fn @changeUserSelectedTask status)}}>{{status.label}}</a>
```

Comment:
This should be a `<button>`.
Refer: https://www.webaxe.org/proper-use-buttons-links/

Reply:
Oh yes, my bad! Missed it